.mailmap

```diff
@@ -0,0 +1,122 @@
+Cynthia Coan <cynthia@coan.dev>
+Cynthia Coan <cynthia@coan.dev>                        <SecurityInsanity@users.noreply.github.com>
+Donny Yung <donaldryung@gmail.com>                     <donaldyung@datawire.io>
+Flynn <emissary@flynn.kodachi.com>                     <flynn+github@kodachi.com>
+Flynn <emissary@flynn.kodachi.com>                     <flynn@datawire.io>
+Flynn <emissary@flynn.kodachi.com>                     <flynn@ryoohko.local>
+Flynn <emissary@flynn.kodachi.com>                     <kflynn@users.noreply.github.com>
+Gabriel Linden Sagula <gsagula@gmail.com>
+Itamar Turner-Trauring <itamar@itamarst.org>           <itamar@datawire.io>
+John Esmet <john.esmet@gmail.com>                      <jesmet@appnexus.com>
+John Esmet <john.esmet@gmail.com>                      <johnesmest@datawire.io>
+John Esmet <john.esmet@gmail.com>                      <johnesmet@datawire.io>
+Luke Shumaker <lukeshu@lukeshu.com>                    <luke@lukeshu.com>
+Luke Shumaker <lukeshu@lukeshu.com>                    <lukeshu@datawire.io>
+Noah Krause <krausenoah@gmail.com>
+Noah Krause <krausenoah@gmail.com>                     <nkrause@MacBook-Pro.local>
+Noah Krause <krausenoah@gmail.com>                     <nkrause@Opss-MacBook-Air.local>
+Noah Krause <krausenoah@gmail.com>                     <nkrause@datawire.io>
+Noah Krause <krausenoah@gmail.com>                     <noahkrause@Noahs-MacBook-Pro.local>
+Philip Lombardi <plombardi89@gmail.com>
+Philip Lombardi <plombardi89@gmail.com>                <893096+plombardi89@users.noreply.github.com>
+Philip Lombardi <plombardi89@gmail.com>                <plombardi@datawire.io>
+Shahriar Rostami <srdsecond@gmail.com>                 <54422255+alphashr@users.noreply.github.com>
+Stephanie Coyle <stephaniecoyle391@gmail.com>
+Stephanie Coyle <stephaniecoyle391@gmail.com>          <55929576+scoyle391@users.noreply.github.com>
+Stephanie Coyle <stephaniecoyle391@gmail.com>          <scoyle@datawire.io>
+
+# Outside contributors (inconsistent email -- did I choosed the correct email?)
+Brandon Catcho <bcatcho@doubledutch.me>                <bcatcho@gmail.com>
+Christoph Grossegger <christoph.grossegger@gmail.com>  <christoph@dust-bit-games.com>
+Dan Sipple <dsipple@etsy.com>                          <sipple.dan@gmail.com>
+Endre Czirbesz <endre.czirbesz@rungway.com>            <endre@czirbesz.hu>
+Endre Czirbesz <endre.czirbesz@rungway.com>            <endrec@users.noreply.github.com>
+Giovanni Gargiulo <gargiulo.gianni@gmail.com>          <giovanni.gargiulo@hbc.com>
+John Morrissey <jmorrissey@devoted.com>                <jwm@devoted.com>
+Josue Diaz <josue_diaz@affirmednetworks.com>           <josuesdiaz15@msn.com>
+Markus Maga <markus@maga.nu>                           <markus@maga.se>
+Niko Kurtti <niko.kurtti@interdax.com>                 <niko@kurtti.eu>
+Phil Peble <philpeble@gmail.com>                       <ppeble@activecampaign.com>
```

Comment:
I think that Phil always contributed as an ActiveCampaign employee. Might be worth checking, he's easy to reach on Slack.

Reply:
OK, I've sent 12 emails of the form

```
Date: Tue, 15 Nov 2022 10:14:33 -0700
Message-ID: <emissary-mailmap-1@lukeshu.com>
From: Luke Shumaker <lukeshu@lukeshu.com>
To: Brandon Catcho <bcatcho@doubledutch.me>, Brandon Catcho <bcatcho@gmail.com>
Subject: Attribution for Emissary/Ambassador contributions
User-Agent: Wanderlust/2.15.9 (Almost Unreal) SEMI-EPG/1.14.7 (Harue)
 FLIM-LB/1.14.9 (Gojō) APEL-LB/10.8 EasyPG/1.0.0
 Emacs/28.2 (x86_64-pc-linux-gnu) MULE/6.0 (HANACHIRUSATO)
MIME-Version: 1.0 (generated by SEMI-EPG 1.14.7 - "Harue")
Content-Type: text/plain; charset=US-ASCII

Hi Brandon,

I'm with Emissary-ingress (formerly Ambassador API Gateway), and am
setting up a Git `.mailmap` file to normalize contributor
information.  I see that there are multiple email addresses that
you've committed with; which address would you prefer to be the
canonical/current one?

  - bcatcho@doubledutch.me
  - bcatcho@gmail.com

-- 
Cheers,
~ Luke Shumaker
```
for these 12 sets of names/addresses
```
Brandon Catcho <bcatcho@doubledutch.me>                <bcatcho@gmail.com>
Christoph Grossegger <christoph.grossegger@gmail.com>  <christoph@dust-bit-games.com>
Dan Sipple <dsipple@etsy.com>                          <sipple.dan@gmail.com>
Endre Czirbesz <endre.czirbesz@rungway.com>            <endre@czirbesz.hu> 
Giovanni Gargiulo <gargiulo.gianni@gmail.com>          <giovanni.gargiulo@hbc.com>
John Morrissey <jmorrissey@devoted.com>                <jwm@devoted.com>
Josue Diaz <josue_diaz@affirmednetworks.com>           <josuesdiaz15@msn.com>
Markus Maga <markus@maga.nu>                           <markus@maga.se>
Niko Kurtti <niko.kurtti@interdax.com>                 <niko@kurtti.eu>
Phil Peble <philpeble@gmail.com>                       <ppeble@activecampaign.com>
Romario Maxwell <maxwellra@jncb.com>                   <romario.maxwell@gmail.com>
Wenhua Zhao <whzhao@gmail.com>                         <wenhuaz@splunk.com>
```